general/odoo_basics/export-data.rst

```diff
@@ -0,0 +1,62 @@
+======================
+Exporting Data in Odoo
+======================
+
+When working with a database, it sometimes appears interesting to have a
+backup of the data. That backup ensures you always have access to them
+in case of minor or major issues with the database. Also, it can help to
+do reporting over your activities (even if Odoo offers a precise and
+easy reporting tool with each available application).
+
+With Odoo, you can export nearly any record. The only thing to do is to
+activate the list view on the items that need to be exported. Then, hit
+the *Action* button and click on *Export*.
+
+.. image:: media/list-view-export.png
+   :align: center
+   :alt: view of the different things to enable/click to export data
+
+Pretty simple, this action still comes with some specificities. In fact,
+when clicking on *Export*, a pop-up window appears with several
+options for the data to import:
+
+.. image:: media/export-data-overview.png
+   :align: center
+   :alt: overview of all the fields to take into account when exporting data in Odoo
+
+1. With the *I want to update data* option ticked, the system only
+   shows the fields which can be imported. This is very helpful in
+   case you want to update existing records. Basically, this works
+   like a filter. Leaving the box unticked gives way more field
+   options because it shows all the fields, not just the ones which
+   can be imported.
+
+2. When exporting, you can choose between two formats: .csv and .xls.
+   With .csv, items are separated with a comma, while .xls separated
+   items with columns.
```

Comment:
it's right to indicate that the csv is separated with a comma as it can be a different separator. However, it doesn't mean that the particularity of xls is to be separated with columns. It's just Microsoft Excel format. csv can be imported in various Softwares that are not used to handle spreadsheets, as xls does.